beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -17,6 +17,30 @@ type WeakSubjectivityResponse struct {
 	} `json:"data"`
 }
 
+type LightClientFinalityUpdateResponseJson struct {
```

Comment:
We understand there is an ongoing efforts to remove `apimiddleware` structs.  

Unfortunately, our original implementation was written before that.   

Please help to give some guidance (or reference PR), and we will be happy to refactor.